### ISSUE TITLE
problem loading celeste atlas

### ISSUE THREAD

**Issue by microlith57** [CelestialCartographers/Loenn#7]
when running `love -fused Lönn.love`, it aborts with
```
! Task Failed:    [string "utils/utils.lua"]:203: Could not decode file 'placeholder.png' to ImageData: unsupported file format
stack traceback:
    [C]: in function 'newImageData'
    [string "utils/utils.lua"]:203: in function 'getCachedDataImage'
    [string "sprite_loader.lua"]:199: in function 'getCacheOrLoadSpriteAtlas'
    [string "atlases.lua"]:45: in function 'loadCelesteAtlas'
    [string "atlases.lua"]:61: in function 'loadCelesteAtlases'
    [string "scenes/loading.lua"]:81: in function 'func'
    [string "utils/tasks.lua"]:191: in function <[string "utils/tasks.lua"]:191>
! Task Failed:    [string "celeste_render.lua"]:352: attempt to index field 'gameplay' (a nil value)
stack traceback:
    [string "celeste_render.lua"]: in function 'func'
    [string "utils/tasks.lua"]:191: in function <[string "utils/tasks.lua"]:191>
Aborted (core dumped)
```
during the loading screen on linux.

**Comment by microlith57** [comment]
the atlas was empty (0 bytes); deleting and regenerating it solved the issue. leaving the issue open for now as broken atlases should be detected instead of crashing

**Comment by Cruor** [comment]
This is fixed in f568b21, should be a part of v0.1.1.
